<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gestion Service</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>781</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Ajouter</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>561</width>
          <height>231</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QTextEdit, QListView {
    background-color: white;
    background-image: url(draft.png);
    background-attachment: fixed;
}</string>
        </property>
        <property name="title">
         <string>Ajout</string>
        </property>
        <widget class="QLabel" name="label_30">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="LineEdit_Nom">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>40</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="ajouter">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>190</y>
           <width>84</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
          </font>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Ajouter</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="quitter_ajouter">
         <property name="geometry">
          <rect>
           <x>380</x>
           <y>190</y>
           <width>84</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Quiter</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_32">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prix</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="LineEdit_Prix">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>80</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_33">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>51</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Qte</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="LineEdit_Qte">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>120</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Modifier</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>571</width>
          <height>231</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Modification</string>
        </property>
        <widget class="QLabel" name="label_36">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>80</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_prix_modif">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>110</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_37">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Prix</string>
         </property>
        </widget>
        <widget class="QPushButton" name="modifier">
         <property name="geometry">
          <rect>
           <x>270</x>
           <y>190</y>
           <width>84</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Modifier</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_38">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>150</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Qte</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_qte_modif">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>150</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="quitter_modifier">
         <property name="geometry">
          <rect>
           <x>370</x>
           <y>190</y>
           <width>84</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Quiter</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_nom_modif">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>80</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_id_modif">
         <property name="geometry">
          <rect>
           <x>90</x>
           <y>50</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_39">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>71</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="Tab_2">
       <attribute name="title">
        <string>Supprimer</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>511</width>
          <height>221</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Suppression</string>
        </property>
        <widget class="QPushButton" name="supprimer">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>180</y>
           <width>84</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>supprimer</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="label_43">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>21</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Id</string>
         </property>
        </widget>
        <widget class="QPushButton" name="quitter_supprimer">
         <property name="geometry">
          <rect>
           <x>350</x>
           <y>180</y>
           <width>84</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Quiter</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_IDS">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>30</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Affichage</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>591</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Affichage</string>
        </property>
       </widget>
       <widget class="QPushButton" name="quitter_afficher">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>220</y>
          <width>84</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
         </font>
        </property>
        <property name="contextMenuPolicy">
         <enum>Qt::NoContextMenu</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
        </property>
        <property name="text">
         <string>Quiter</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="afficher">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>220</y>
          <width>84</width>
          <height>28</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
         </font>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
        </property>
        <property name="text">
         <string>afficher</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Tri</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>761</width>
          <height>221</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="title">
         <string>Tri</string>
        </property>
        <widget class="QRadioButton" name="radioButton_TRInProd">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>82</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Tri Nom</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="radioButton_TRInProdPrix">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>30</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Tri qte</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_AjoutProdui_9">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>130</y>
           <width>84</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Quiter</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QTableView" name="tableView_tri">
         <property name="geometry">
          <rect>
           <x>200</x>
           <y>20</y>
           <width>551</width>
           <height>191</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Recherche</string>
       </attribute>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>761</width>
          <height>251</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Reference Sans Serif</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Recherche</string>
        </property>
        <widget class="QLineEdit" name="ProduitNom_rechecher">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>60</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>60</y>
           <width>31</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel 
{
rgb(255, 255, 255)
font: 8pt &quot;MS Shell Dlg 2&quot;;}</string>
         </property>
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
        <widget class="QPushButton" name="rechercher_q">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>160</y>
           <width>84</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>Quiter</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="rechercher">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>120</y>
           <width>84</width>
           <height>28</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Reference Sans Serif</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgb(108, 135, 255));

}</string>
         </property>
         <property name="text">
          <string>rechercher</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QTableView" name="tableView_affirechercher">
         <property name="geometry">
          <rect>
           <x>160</x>
           <y>20</y>
           <width>521</width>
           <height>201</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_7">
       <attribute name="title">
        <string>Statistique</string>
       </attribute>
       <widget class="QFrame" name="aaaaa">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>10</y>
          <width>751</width>
          <height>391</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>17000</width>
          <height>17000</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_31"/>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
